src/views/examples/Register.js

```diff
@@ -24,145 +25,83 @@ import {
   InputGroupText,
   InputGroup,
   Col,
-  FormFeedback,
 } from "reactstrap";
 
 
+
 const Register=()=> {
-  
-  const [password, setPassword]= useState('');
-  const [confirmPassword, setConfirmPassword]=useState('');
-  const [pass, passState]=useState('');
-  const [email, setEmail] = useState("");
-  const [emailError, setEmailError] = useState('')
-  const [isDisabled, setIsdisabled]= useState(true)
-
-  const validateEmail = async (e) => {
-    var class_valid = "is-valid form-control";
-    var class_invalid = "is-invalid form-control";
-    
-    if (
-      /^[a-zA-Z0-9.!#$%&'*+/=?^_`{|}~-]+@iitk.ac.in/.test(e.value)
-    ){
-      setEmail(e.value);
-      setEmailError("valid email");
-      e.className=class_valid;
-    }
-    else{
-      setEmailError("invalid iitk email");
-      e.className=class_invalid;
-    }
-  }
-  
-  const onRegister = async (e)=> {
-   
-    if(emailError==="valid email"&& password==confirmPassword){
-      if (e) e.preventDefault();
+
+  const [roll, setRoll] = useState("");
+  const history = useHistory();
+
+
+  const onSubmit = async (e)=> {
+    const id=1234;
+    history.push("/auth/setPass/access/:id")
```

Comment:
you can remove this now and check by changing URL in backend for mail it should match prefix of current URL